packages/nuxt/src/app/composables/chunk.ts

```diff
@@ -0,0 +1,48 @@
+import { useNuxtApp } from '#app/nuxt'
+
+export interface ReloadNuxtAppOptions {
+  /**
+   * Number of milliseconds in which to ignore future reload requests
+   *
+   * @default {10000}
+   */
+  ttl?: number
+  /**
+   * Force a reload even if one has occurred within the previously specified TTL.
+   *
+   * @default {false}
+   */
+  force?: boolean
+  /**
+   * The path to reload. If this is different from the current window location it will
+   * trigger a navigation and add an entry in the browser history.
+   *
+   * @default {window.location.pathname}
+   */
+  path?: string
+}
+
+export function reloadNuxtApp (options: ReloadNuxtAppOptions = {}) {
+  const path = options.path || window.location.pathname
```

Comment:
I guess we can add this to optimizations to be treeshaked from server bundle or directly protect with `if (process.server) { return }`

Reply:
```suggestion
  if (process.server) { return }
  const path = options.path || window.location.pathname
```